buildenv/jenkins/jobs/infrastructure/copyrightCheckDir.groovy

```diff
@@ -1,6 +1,6 @@
 /*
  * ===========================================================================
- * (c) Copyright IBM Corp. 2019, 2023 All Rights Reserved
+ * (c) Copyright IBM Corp. 2019, 2024 All Rights Reserved
```

Comment:
While you're here, could you fix the indentation of line 23, please?

Reply:
... and remove the extra blank line at the end of this file?

Reply:
Updated.